src/site/stages/build/process-cms-exports/schemas/output/media-image.js

```diff
@@ -1,20 +1,25 @@
 module.exports = {
   type: 'object',
   properties: {
-    entityType: { type: 'string', enum: ['media'] },
-    entityBundle: { type: 'string', enum: ['image'] },
-    name: { type: 'string' },
-    thumbnail: { $ref: 'output/file' },
-    entityMetatags: { $ref: 'MetaTags' },
-    fieldMediaInLibrary: { type: 'boolean' },
-    fieldOwner: { $ref: 'output/taxonomy_term-administration' },
-    image: { $ref: 'output/file' },
+    entity: {
```

Comment:
:thinking: Does it make sense to wrap this in an `entity` object in the `media-image` transformer, or in the parent's transformer? For this one entity, I don't think I have a preference, but for consistency, we should probably always do it in the parent transformer.

Reply:
Yeah I think it makes sense to wrap it in the parent transformer. I could see the same issue with `node-health_care_region_page` happening, or something similar. Wrapping it in it's parent transformer gives more flexibility. And also it's best for consistency like you said 👍.

Reply:
I refactored the media transformers in this [commit](https://github.com/department-of-veterans-affairs/vets-website/pull/13962/commits/6e0744cad4706ee01da840872cb894fdd84a97fa).